### PR TITLE
Broken link 2

### PR DESCRIPTION
**Pull request by antonilol**
test gh ci

### CODE REVIEW

File: 03-transactions.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This details the exact format of on-chain transactions, which both sides need to
         * [Commitment Transaction Outputs](#commitment-transaction-outputs)
           * [`to_local` Output](#to_local-output)
           * [`to_remote` Output](#to_remote-output)
-          * [`to_local_anchor` and `to_remote_anchor`](#to_local_anchor-and-to_remote_anchor-output-option_anchor_outputs)
+          * [`to_local_anchor` and `to_remote_anchor`](#to_local_anchor-and-to_remote_anchor-output-option_anchors)
```
Comment (vincenzopalazzo) [inline 03-transactions.md:15]: have you tried to remove this and check if the CI will fail?

Comment (antonilol) [reply 03-transactions.md:15]: e0f6d5b

Comment (antonilol) [reply 03-transactions.md:15]: it also did not catch this one

Comment (vincenzopalazzo) [reply 03-transactions.md:15]: Amazing a url checker that is not able to check the relative url! so maybe you can run your scrip inside the CI?

**Pull request closed without merging**